src/main/kotlin/pullADRs.kt

```diff
@@ -2,10 +2,7 @@ package uk.gov.justice.hmpps.architecture
 
 import com.structurizr.Workspace
 import com.structurizr.documentation.AdrToolsImporter
-import com.structurizr.model.SoftwareSystem
-import org.eclipse.jgit.api.Git
 import uk.gov.justice.hmpps.architecture.annotations.ADRSource
-import java.io.File
 
 fun pullADRs(workspace: Workspace) {
   val systemsWithADRs = workspace.model.softwareSystems.stream().filter { ADRSource.getFrom(it) != null }
```

Comment:
I think this can be collapsed into a single chain

Reply:
done in 6f21e4af28371704702e0edca372bcdcc13702a1